### PR TITLE
Usgs/hazdev geoserve ws#8

### LINKED ISSUE #8: implement factory
- getPlaces(lat, lon, limit=5, featureCode='P', minPopulation=0, maxRadiusKm=null)

Methods to be implemented later:
- getAuthoritativeNetwork(lat, lon)
- getFeRegion(lat, lon)
- getRegion(lat, lon)
- getTectonicSummary(lat, lon)

### PR DESCRIPTION
**Pull request by jmfee-usgs**
getPlaces() is only implemented method, issues will be added for other methods.

Closes #8 .

### CODE REVIEW

File: src/htdocs/index.php [hunk 1]
```diff
@@ -1,10 +1,107 @@
...
+  include_once '../conf/config.inc.php';
+
+  $TITLE = "Nearby Places";
   include_once 'template.inc.php';
 }
 
+$latitude = param('latitude', null);
+$longitude = param('longitude', null);
+$limit = param('limit', 5);
+if (!$limit) {
```
Comment (ehunter-usgs) [inline src/htdocs/index.php]: param() will set $limit to null

File: src/htdocs/index.php [hunk 2]
```diff
@@ -1,10 +1,107 @@
...
+  $limit = null;
+}
+$distance = param('distance', null);
+if (!$distance) {
```
Comment (ehunter-usgs) [inline src/htdocs/index.php]: param() will set $distance to null

File: src/htdocs/index.php [hunk 3]
```diff
@@ -1,10 +1,107 @@
...
+  $distance = null;
+}
+$population = param('population', null);
+if (!$population) {
```
Comment (ehunter-usgs) [inline src/htdocs/index.php]: param() will set $population to null

File: src/htdocs/index.php [hunk 4]
```diff
@@ -1,10 +1,107 @@
...
+    <button>Search</button>
+  </div>
+</form>
+
+<?php
+
+if ($latitude !== null && $longitude !== null) {
+  $options = array();
+  if ($distance !== null) {
+    $options['distance'] = $distance * 1000;
```
Comment (ehunter-usgs) [inline src/htdocs/index.php]: why are we converting to meters?

File: src/htdocs/index.php (hunk 1 above)
Comment (jmfee-usgs) [inline src/htdocs/index.php]: This handles the case when the form field is cleared, the library sets a default limit.  Maybe we should remove the default.

File: src/htdocs/index.php (hunk 2 above)
Comment (jmfee-usgs) [inline src/htdocs/index.php]: When you submit the form without a value, it is the empty string instead of null.

File: src/htdocs/index.php (hunk 4 above)
Comment (jmfee-usgs) [inline src/htdocs/index.php]: St_distance returns meters.  We could accept meters as input.

This was more of a quick demo to show the factory working, than a finished search form or web service interface.  Those portions can and should change.

**Pull request merged**